packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeVisualizer.ts

```diff
@@ -215,45 +252,93 @@ async function visualizeObjectNode(
  */
 async function visualizeMapNode(
 	tree: VerboseTreeNode,
-	nodeSchema: SimpleMapNodeSchema,
-	treeSchema: SimpleTreeSchema,
+	schema: SimpleMapNodeSchema,
+	treeDefinitions: ReadonlyMap<string, SimpleNodeSchema>,
+	{ allowedTypes, isRequired }: FieldSchemaProperties,
+	visualizeChildData: VisualizeChildData,
+): Promise<VisualSharedTreeNode> {
+	const mapNodeSchemaProperties: Record<string, FieldSchemaProperties> = {};
+	for (const key of Object.keys(tree.fields)) {
+		mapNodeSchemaProperties[key] = {
+			allowedTypes: schema.allowedTypes,
+			isRequired: undefined,
```

Comment:
Nit: a quick comment here might be nice for future maintainers. Maybe something like:
```suggestion
			// Map values are always required. Don't display field requirement information, since that information is redundant.
			isRequired: undefined,
```